src/malcolmWidgets/method/methodPlot.component.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTheme } from '@material-ui/core/styles';
+import Plot from 'react-plotly.js';
```

Comment:
This will affect the code splitting, we need to look at how to dynamically import this and look at how to link it into the other attribute plot.